saleor/graphql/warehouse/dataloaders.py

```diff
@@ -347,7 +347,8 @@ class StockByIdLoader(DataLoader):
     context_key = "stock_by_id"
 
     def batch_load(self, keys):
-        return Stock.objects.using(self.database_connection_name).in_bulk(keys).values()
+        stocks = Stock.objects.using(self.database_connection_name).in_bulk(keys)
+        return [stocks.get(key) for key in keys]
```

Comment:
Shouldn't be `.get(key, [])`?

Reply:
IMO I think we shouldn't use default here. What code should do if receives `[]` instead of `Stock` object. ?

Reply:
We could use `[]` as the default in dataloaders like `AvailableShippingMethodByCheckoutId` when we return list of objects.

Reply:
I agree with @fowczarek